starfish/test/full_pipelines/cli/test_iss.py

```diff
@@ -145,25 +144,13 @@ class TestWithIssData(unittest.TestCase):
                 "$['codebook']",
             ),
             "-o", lambda tempdir, *args, **kwargs: os.path.join(
-                tempdir, "results", "decoded-spots.nc"),
+                tempdir, "results", "spots.nc"),
             "PerRoundMaxChannelDecoder",
         ],
     )
 
-    def test_run_pipeline(self):
-
-        tempdir = exec.stages(
-            TestWithIssData.STAGES,
-            TestWithIssData.SUBDIRS,
-            keep_data=True,
-        )
-
-        try:
-            intensities = IntensityTable.load(os.path.join(tempdir, "results", "decoded-spots.nc"))
-            genes, counts = np.unique(
-                intensities.coords[Features.TARGET], return_counts=True)
-            gene_counts = pd.Series(counts, genes)
-            assert gene_counts['ACTB_human'] > gene_counts['ACTB_mouse']
-        finally:
-            if os.getenv("TEST_ISS_KEEP_DATA") is None:
-                shutil.rmtree(tempdir)
+    def verify_results(self, intensities):
+        genes, counts = np.unique(
+            intensities.coords[Features.TARGET], return_counts=True)
+        gene_counts = pd.Series(counts, genes)
+        assert gene_counts['ACTB_human'] > gene_counts['ACTB_mouse']
```

Comment:
Sigh, this assertion is pretty worthless (p ~ 0.5 that it catches an error :wink:). Can we add a TODO to make this a stronger test? 

Also I think I created a TEST dataset to replace these data. Could you slot that in instead of these data? If you have to swap the sign on the assertion you have my blessing.